Update process. (#270)
solid/process: README.md

This repository details how changes to Solid may be proposed and accepted.

# Solid Specification

> The following describes how changes to the specifications in the Solid ecosystem may be proposed and accepted. Anyone may submit a proposal to alter this process; such proposals will be reviewed only by [Tim Berners-Lee](https://github.com/timbl), who is the Solid Director.

Reviewer: Is there a document or pointer for these changes?

The "will be reviewed only" is a bit strong, and unenforceable; I expect there will be reviews by multiple people. Do we mean that Tim is the only approver?

Language-wise, I would just use "[a/the] Solid Director" to avoid redundancy.

Reviewer 2: > The "will be reviewed only" is a bit strong, and unenforceable; I expect there will be reviews by multiple people. Do we mean that Tim is the only approver?

Agree with @RubenVerborgh here - maybe something like the following would work better?

```suggestion
The following describes how changes to the specifications in the Solid ecosystem may be proposed and accepted. Anyone may submit a proposal to alter this process; such proposals may be reviewed by members of the Solid team, and must be approved by the [Solid Director](https://github.com/timbl).
```

## Editorial Team

The Solid [Editorial Team](https://github.com/orgs/solid/teams/editors) is responsible for the implementation of the Solid Specification process. Members of the Editorial Team are appointed by the Solid Director. The Editorial Team is comprised of all the Editors appointed by the Solid Director, who are listed at [editors.md](editors.md), along with their assignments, contact details, and affiliations. The Editorial Team shepherds Solid and coordinates with those who participate in [Solid Panels](#solid-panels) and with [Stakeholders](#stakeholders) and [Implementers](#implementers). The Solid Director may also appoint support personnel on an as-needed basis, such as a Release Manager, for organizing the workflow of the specification process.

Anyone may apply to be an Editor, and must include one or more requested editorial assignments as part of their application. These requests are reviewed only by the Solid Director. Editor applications that can demonstrate the support of a relevant panel or group of community members will be favorably considered.

## Contributors

> Any individual who has been involved in proposals to improve the [Solid Specification](https://github.com/solid/specification) has provided a valuable service to the [Solid Project](https://www.solidproject.org) and is encouraged to continue.

Reviewer 2: Yes that was exactly the rationale at the time. I think it (and similar statements in that vein) can be safely removed now for the sake of making the document as concise as possible.

Contributions are welcome, including identifying problems, asking questions, and proposing normative changes.

There are many topics, problems, or ideas best addressed by a group of contributors with specific domain expertise in [Solid Panels](#solid-panels).

### Solid Panels

> Solid Panels are groups of contributors focused on a specific problem or domain relevant to Solid, with an aim to propose changes to the [Solid Specification](https://github.com/solid/specification). Domains may be technical, non-technical, or some combination of the two. For example, a Security Panel could focus on the evaluation and advancement of the Solid security model. Only those who have agreed to the [W3C Community Contributor License Agreement](https://www.w3.org/community/about/agreements/cla/) may participate in a Solid Panel.

Reviewer 2: I think we're missing a pathway for how and where the agreement to that W3C CLA takes place. For example, in [Solid Specification](https://github.com/solid/specification/) there isn't a CONTRIBUTING document that implies the work is being done under the W3C CLA, nor language to that effect in the main README, which seems to be the common approach in other W3C endeavors. The [Shape Trees Specification](https://github.com/shapetrees/specification/) is an example where we aimed to mirror the pattern common in other W3C specification repos - with a [CONTRIBUTING](https://github.com/shapetrees/specification/blob/main/CONTRIBUTING.md) document and pointer from the main README. We may want to do something similar in the main specification repo and the panels.

1. When a Panel starts drafting the text of a proposal, the Panel should notify the Editors.

The above is not required and may be superfluous for small changes, but is likely to speed up the overall process for larger issues.

An Editor determines whether a Candidate Proposal includes a substantive change and marks it accordingly. If there is any disagreement among Editors, the Candidate Proposal will be automatically marked as including a substantive change.

When a proposal is first submitted, the Solid Editors will assign a single Editor as the shepherd for the proposal and assign a time window for review and its acceptance or rejection; the time window will be either 1 or 2 months from the date of proposal submission, depending on the complexity of the proposal. If an Editor is a contributor to the proposal, they will not be assigned as the proposal's shepherd. This time window includes all discussion and revision that may be needed to improve the proposal. The Solid Director, working with the Solid Editors and any administrative support personnel, will maintain a dashboard of open and closed proposals, time windows for review, and review status.

> If a contributor disapproves of Candidate Proposal that is under consideration, they must provide substantive, written comments that contain proposed remedies to the Solid Editors at least 1 week before the end of the review time window. The Solid Editors will consider the comments and suggest possible remedies.

Reviewer 2: ```suggestion
If a contributor disapproves of a Candidate Proposal that is under consideration, they must provide substantive, written comments that contain proposed remedies to the Solid Editors at least 1 week before the end of the review time window. The Solid Editors will consider the comments and suggest possible remedies.
```
